apps/chat/README.md

```diff
@@ -9,7 +9,20 @@ The easiest way to get started with messaging in the Amazon Chime SDK is to depl
 - An [AWS account](https://aws.amazon.com/free/) that provides access to AWS services.
 - [Node.js](https://nodejs.org/en/download/) version 14 or higher.
 
-## Deploying the solution
+## Deploying the demo app
+
+You can run deploy.sh script in this folder to automatically deploy the demo app and it's backend.
+
+```bash
+cd apps/chat
+./deploy.sh
+```
+
+Set the parameters in the `Config.js` based on the stack outputs and redeploy one more time.
```

Comment:
`redeploy one more time` is confusing. Is there a way to automate this process in a single deploy command? For example, you may put the stack outputs to a separate JSON file and automatically re-deploy (or deploy only the required part) using that JSON file.

Reply:
yup, updated. see next rev.